lib/pf/SwitchFactory.pm

```diff
@@ -165,6 +179,36 @@ sub config {
     return \%temp;
 }
 
+sub getModule {
+    my ($type) = @_;
+    return (exists $TYPE_TO_MODULE{$type} ) ? $TYPE_TO_MODULE{$type} : undef;;
+}
+
+=item buildVendorsList
```

Comment:
I think we should discuss the new sub headers all of us together before we start putting them in the codebase because then we may end up with (an even more) heterogenous pod documentation.

So I'd remove the usage + return for now.